task/t2_pandas.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+
+FILE_PATH = "f1.csv"
+data_frame = pd.read_csv(FILE_PATH)
+
+MAX_TEMP_COL = 'Max TemperatureC'
+MIN_TEMP_COL = 'Min TemperatureC'
```

Comment:
move constants to a constants.py file and import these variables from there

Reply:
sir ap kiya kehna cha rhy hn samj nhi i